### DISCUSSION TITLE
Laptop screen refresh rate is only 60Hz

### DISCUSSION THREAD

**Discussion by ghost** [seerge/g-helper#384] [Q&A] [answered]
I used G-Helper for a period of time. When I started using this software, I thought it was very powerful, but I don't know why. Recently, I found that the highest screen on my laptop is only 60Hz. No matter which mode I choose.
My model is ASUS TUF Gaming F15 FX506HE_FX506HE
how can i solve it?
Thank you so much!

**Comment by seerge** [comment]
@kurai23 can you select higher refresh rate in windows screen settings?

**Reply by ghost** [reply]
The refresh rate in Windows screen settings is 165hz

**Comment by ghost** [comment]
Part of the log:
2023/5/15 18:10:18: Screen = 60Hz : -2
2023/5/15 18:10:18: ScreenOverdrive = 0 : OK
2023/5/15 18:10:18: Screen = 60Hz : -2
2023/5/15 18:10:18: ScreenOverdrive = 0 : OK
2023/5/15 18:10:19: Screen = 60Hz : -2
2023/5/15 18:10:19: ScreenOverdrive = 0 : OK
2023/5/15 18:10:19: Screen = 60Hz : -2
2023/5/15 18:10:19: ScreenOverdrive = 0 : OK
2023/5/15 18:10:19: Screen = 60Hz : -2
2023/5/15 18:10:19: ScreenOverdrive = 0 : OK
2023/5/15 18:10:20: Screen = 60Hz : -2
2023/5/15 18:10:20: ScreenOverdrive = 0 : OK
2023/5/15 18:10:20: Screen = 60Hz : -2
2023/5/15 18:10:20: ScreenOverdrive = 0 : OK
2023/5/15 18:10:20: Screen = 60Hz : -2
2023/5/15 18:10:20: ScreenOverdrive = 0 : OK
2023/5/15 18:12:07: Screen = 60Hz : -2
2023/5/15 18:12:07: ScreenOverdrive = 0 : OK
2023/5/15 18:12:07: Screen = 165Hz : OK
2023/5/15 18:12:07: ScreenOverdrive = 1 : OK
2023/5/15 18:12:08: Screen = 60Hz : -2
2023/5/15 18:12:08: ScreenOverdrive = 0 : OK
2023/5/15 18:12:08: Screen = 165Hz : OK
2023/5/15 18:12:08: ScreenOverdrive = 1 : OK
2023/5/15 18:12:19: Screen = 165Hz : OK
2023/5/15 18:12:19: ScreenOverdrive = 1 : OK
2023/5/15 18:12:22: Screen = 60Hz : -2
2023/5/15 18:12:22: ScreenOverdrive = 0 : OK
2023/5/15 18:12:25: Screen = 165Hz : OK
2023/5/15 18:12:25: ScreenOverdrive = 1 : OK
2023/5/15 18:12:28: Screen = 60Hz : -2
2023/5/15 18:12:28: ScreenOverdrive = 0 : OK
2023/5/15 18:12:28: Screen = 60Hz : -2
2023/5/15 18:12:28: ScreenOverdrive = 0 : OK
The 'frequency' of the config.json file remains unchanged after each switch

**Reply by ghost** [reply]
Sorry, I used a translator and may not be able to express myself clearly

**Reply by seerge** [reply]
But here it says that it succeded settings 165Hz : ``2023/5/15 18:12:07: Screen = 165Hz : OK`` what happens if you click 165hz button ?

**Reply by ghost** [reply]
2023/5/15 18:33:41: Screen = 165Hz : OK 2023/5/15 18:33:41: ScreenOverdrive = 1 : OK
But the actual screen can only use 60Hz

**Reply by ghost** [reply]
After reinstalling Armoury Crate, the screen can recover to 165hz

**Reply by seerge** [reply]
@kurai23 can "recover" in g-helper or in windows? You just mentioned in a previous message that it could go to 165hz anyway ?

**Reply by ghost** [reply]
Install Armory Crate and close G-helper
[https://www.asus.com/supportonly/armoury%20crate/helpdesk_knowledge/](url)
I don't quite understand your reply

**Reply by ghost** [reply]
And, I clicked on the 60 button, but the gray selection box is still on the 165 button. Is this normal
I was unable to upload the image, so I used the URL
[https://imgse.com/i/p9gz4k4](url)

**Comment by ghost** [comment, marked as answer]
Solved the issue of graphics card driver updates. Sorry to bother you
Thank you very much for your answer. Thank you very much

**Reply by seerge** [reply]
@kurai23 ah, ok! glad it worked out.
